### ISSUE TITLE
submissions / export : "Failed to generate export"

### ISSUE THREAD

**Issue by oopen** [ohmyform/ohmyform#152]
#### What's this issue about?
Export submissions didn't work.
No requets is send, and display error message

#### Expected behavior

Export submissions to file (download)

#### Actual behavior
Nothing only error in UI

#### Steps to reproduce
Create a form, answer to it, go to submissions, clic export

#### Environment
Docker, minimal compose file

![Capture d’écran de 2022-02-05 11-11-48](https://user-images.githubusercontent.com/7348132/152638040-ac1befa6-ece8-40f8-9bed-88ca84be7800.png)

**Comment by fiqrimasngot** [comment]
![image](https://user-images.githubusercontent.com/39836635/153720567-8e5af163-0cb0-4338-8b07-940fdfbd568c.png)
 Got the same error.

**Comment by indicozy** [comment]
Same issue, here's my console log:
```
error TypeError: Cannot read properties of undefined (reading 'data')
    at export.submission.action.tsx:80:17
    at s (runtime.js:45:40)
    at Generator._invoke (runtime.js:274:22)
    at Generator.S.forEach.e.<computed> [as next] (runtime.js:97:21)
    at I (submissions-544cb9e4ed8c28b5.js:1:4905)
    at a (submissions-544cb9e4ed8c28b5.js:1:7096)
```

**Comment by DamianoP** [comment]
Same error

**Comment by wodka** [comment]
should be fixed in 1.0.0
